cypress/integration/actNowPage.spec.js

```diff
@@ -0,0 +1,23 @@
+describe("Act Now page", () => {
+  it("renders Act Now page", () => {
+    cy.visit("/");
+    cy.get("[data-cy=navigation-buttons]").contains("Act Now").click();
```

Comment:
Have we modified the navigation buttons to contain `data-cy=navigation-buttons`? This fails when I run locally.

Reply:
Yes. The fail is probably from a timeout issue